bot/kodiak/queue.py

```diff
@@ -286,7 +293,9 @@ async def enqueue_for_repo(
         """
         queue_name = get_merge_queue_name(event)
         transaction = await self.connection.multi()
-        await transaction.sadd(MERGE_QUEUE_NAMES, [queue_name])
+        merge_queues_by_install = f"merge_queue_by_install:{event.installation_id}"
```

Comment:
how hard would it be to add tests for these lines?

Reply:
queue.py is difficult to test in its current state